cmd/jujud/agent/machine/manifolds.go

```diff
@@ -954,6 +979,14 @@ func clockManifold(clock clock.Clock) dependency.Manifold {
 	}
 }
 
+// ifResponsible wraps a manifold such that it only runs if the
+// responsibility flag is set.
+var ifResponsible = engine.Housing{
```

Comment:
What is the purpose of this? For workers where we need one per model, they don't go in the machine manifolds, but in `model/manifolds.go`.

Reply:
Right, I didn't know what `ifResponsible` was and if I needed it. I think there is a bit of knowledge gap around these decorators...